packages/admin-ui-core/src/__tests__/styles.test.ts

```diff
@@ -137,7 +137,7 @@ describe('styles', () => {
     })
   })
 
-  describe('__EXPERIMENTAL__ responsive scales', () => {
+  describe.skip('__EXPERIMENTAL__ responsive scales', () => {
```

Comment:
I don't have enough context on how things work to properly review this PR, but I'm curious about this line. Is this test not needed anymore? Can we remove it instead of skipping?

Reply:
Got it, thanks!